ui/admin/src/assets/i18n/fr.json

```diff
@@ -10,7 +10,7 @@
   "ITEM_ON_LOAN": "en prêt",
   "ITEM_RETURNED": "retourné",
   "ITEM_IN_TRANSIT_FOR_PICKUP": "en transit pour retrait",
-  "ITEM_IN_TRANSIT_TO_HOUSE": "en transit à son origine",
+  "ITEM_IN_TRANSIT_TO_HOUSE": "ITEM_IN_TRANSIT_TO_HOUSE",
```

Comment:
why this is not translated?

Reply:
I have translated it in English, French and German.